dev/view.h

```diff
@@ -39,11 +39,11 @@ namespace sqlite_orm {
             view_t(storage_type& stor, decltype(connection) conn, Args&&... args_) :
                 storage(stor), connection(std::move(conn)), args{std::make_tuple(std::forward<Args>(args_)...)} {}
 
-            size_t size() {
+            size_t size() const {
```

Comment:
why did you add right `const`? right `const` means that nothing that is not mutable is changed. But `size` and `empty` calls access database so these calls can open a file and change opened files OS'es configuration.

Reply:
Sorry! I didn't recognize that it's accessing the database...